commands/connection_pooling.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+const host = require('../lib/host')
+
+function * run (context, heroku) {
+  const util = require('../lib/util')
+  const fetcher = require('../lib/fetcher')(heroku)
+  const {app, args, flags} = context
+
+  let db = yield fetcher.addon(app, args.database)
+  let addon = yield heroku.get(`/addons/${encodeURIComponent(db.name)}`)
+  let credential = `${flags.credential || 'default'}`
```

Comment:
This works but the `flags.credentials` is going to be a string anyway, so you could do `flags.credential || 'default'` without wrapping in a template string.